packages/protocol/contracts/thirdparty/AddressManager.sol

```diff
@@ -65,18 +65,20 @@ contract AddressManager is OwnableUpgradeable {
 
     /**
      * Changes the address associated with a particular name.
-     * @param _name String name to associate an address with.
-     * @param _address Address to associate with the name.
+     * @param name String name to associate an address with.
+     * @param newAddress Address to associate with the name.
      */
     function setAddress(
-        string memory _name,
-        address _address
+        string memory name,
+        address newAddress
     ) external onlyOwner {
-        bytes32 nameHash = _getNameHash(_name);
+        bytes32 nameHash;
+        assembly {
+            nameHash := keccak256(add(name, 32), mload(name))
```

Comment:
I still prefer `nameHash = keccak256(bytes(name))` for readability, as this setAddress won't be used much.

Reply:
lets use `keccak256(bytes(name)`, Jeff. 18 gas cheaper and more readable.